### PR TITLE
fixes(188): Run builder cleanup if SCL for KojiReleaser

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: src/tito/release/main.py [hunk 1]
```diff
@@ -527,6 +527,7 @@ def _koji_release(self):
                 if scl:
                     builder = copy.copy(self.builder)
                     builder.scl = scl
+                    builder.cleanup()
```
Comment (domcleal) [inline src/tito/release/main.py]: I'm unsure if calling cleanup could leave the *original* builder (self.builder) in a strange state if it has already been used?  (The source dir etc might have been cleaned, but self.builder thinks it's dirty still.)

Comment (ehelms) [inline src/tito/release/main.py]: Sorta ad-hoc testing changes to start with, this worked for me originally for scl and non-scl, but @jlsherril is reporting failures after adding the package to say a whitelist in the first non-scl releaser.

**Comment by ehelms** [comment]
Updated to try a different approach, this may be skirting or overlooking the actual issue but it appeared to fix the issue for me in all cases.

**Comment by jlsherrill** [comment]
Can't comment on the correctness of the code, but it appears to solve the issue for me

**Comment by dgoodwin** [comment]
Merging, thanks.

**Pull request merged**